doc/features.rst

```diff
@@ -17,7 +17,7 @@ ________
   
 * Supports a truly reproducible workflow by minimizing the code needed for analysis and visualization.
 * Already used in a variety of research projects, from conception to final publication.
-* All HoloViews objects can be pickled and unpickled.
+* All HoloViews objects can be pickled and unpickled, with no plotting-library dependencies.
```

Comment:
Sure, hopefully styles will normally unpickle also; we just have to make clear that in some cases there might be an object used as a value in those styles whose class is defined by the plotting library, and thus that particular option won't be able to be restored without the plotting library (nor would it presumably make any sense in that case).